xyz.stuerz.BilligSweeper.yml

```diff
@@ -0,0 +1,20 @@
+app-id: xyz.stuerz.BilligSweeper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: billig-sweeper
+rename-desktop-file: billig-sweeper.desktop
+rename-icon: billig-sweeper
+
+modules:
+  - name: billig-sweeper
```

Comment:
you didn't install the metainfo.xml so this build will fail.

Reply:
I think, I've fixed it now.